lib/miscellaneous-settings.js

```diff
@@ -3,7 +3,7 @@
 import { ItemImageSettings } from '../module/actor/actor-sheet.js'
 import { TaggedModifierSettings } from '../module/actor/effect-modifier-popout.js'
 import ModifierBucketJournals from '../module/modifier-bucket/select-journals.js'
-import { QuickRollSettings } from '../module/token-hud.js'
+import { QuickRollSettings } from '../module/token/hud.js'
```

Comment:
We don't have to do this right now, but I would like each module to instantiate its own system settings instead of lumping them all here. That might imply a naming scheme (such as `<module-name>-<setting-name>` to ensure uniqueness. And that might mean a migration between the old schema and the new one. Let me know if you disagree with this or have another approach, or if you just want to keep adding to this catch-all file.

Reply:
Raised this as separate issue #2262